data/categories.txt

```diff
@@ -2814,7 +2815,7 @@ highway-primary|highway-primary_link|highway-residential|highway-secondary|highw
 en:street|st|str|ave|road|rd
 ru:улица|переулок|проезд|тракт|проспект|ул
 uk:вулиця|провулок|проїзд|проспект|бульвар|вул
-de:Strasse|Weg|Avenue|Gasse|str
+de:Strasse|Weg|Avenue|Gasse|Str
```

Comment:
@vng @ygorshenin @mpimenov Does it fit well with our search_string_utils values? Should we store all synonyms in one place only instead of two?